### PR TITLE
LPP 49058 - Wrong link in Asset Publisher configuration

### PR DESCRIPTION
**Pull request by henriquelimadotcom**
Ticket: https://issues.liferay.com/browse/LPS-186036
@dacousalr

### CODE REVIEW

**Comment by henriquelimadotcom** [comment]
@dacousalr Update

**Comment by dacousalr** [comment]
Thank you, @henriquelimadotcom!

It looks good to me. However, since this is going to be a solution to be merged in Master, could you please create the LPS ticket and resend this PR under that LPS?

Thanks!

File: learn-resources/portlet-configuration-web.json
Comment (dacousalr) [inline learn-resources/portlet-configuration-web.json:1]: The following locales should be added specifying language and country, that is, like in https://github.com/liferay/liferay-portal/blob/master/learn-resources/asset-taglib.json:
- ar_SA
- ca_ES
- de_DE
- en_US
- es_ES
- eu_ES
- fi_FI
- fr_FR
- hu_HU
- ja_JP
- nl_NL
- pt_BR
- sv_SE
- zh_CN

**Comment by dacousalr** [comment]
Hi @henriquelimadotcom,

This PR follows the right approach, but some changes are needed:
1. Rename the commit to replace the prefix LPP-49058 with the prefix LPS-186036
2. Follow the indications of this [comment](https://github.com/dacousalr/liferay-portal/pull/353#discussion_r1211581223) paying attention to the locales' codes and which locales need to be changed (it is not the same the Chinese from China zh_CN than the Chinese from Taiwan zh_TW)

Please, let me know if further indications or clarifications are needed.

Best,
Daniel

**Pull request closed without merging**